src/components/InventoryManager/Sidebar/PriceMenu.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Button, Grid, TextField, Typography,
+} from '@material-ui/core';
+import { green } from '@material-ui/core/colors';
+
+const useStyles = makeStyles({
```

Comment:
Could you make the min/max fonts a little smaller so that the entirety of max is visible?